ui-components/src/page_components/listing/ListingsList.tsx

```diff
@@ -29,8 +29,11 @@ const ListingsList = (props: ListingsProps) => {
     }
 
     let unitSummaries = [] as GroupedTableGroup[]
-    if (listing.unitsSummarized !== undefined) {
-      unitSummaries = getSummariesTable(listing.unitsSummarized.byUnitTypeAndRent)
+    if (listing.unitsSummary !== undefined && listing.unitsSummary.length > 0) {
+      unitSummaries = getSummariesTable2(listing.unitsSummary)
+    } else {
+      listing.unitsSummarized !== undefined &&
```

Comment:
Can we move this check (`listing.unitsSummarized !== undefined`) into an `else if (...)`? Just to help readability